challenge/src/App.jsx

```diff
@@ -11,6 +11,7 @@ class App extends Component {
     }
     render() {
         return (
+            console.log("Redered successfully! I think?"),
```

Comment:
@man1han this looks a little wonky to me...
why are you returning the result of console.log with the rendered function